packages/blunderbuss/src/blunderbuss.ts

```diff
@@ -143,6 +154,7 @@ export function blunderbuss(app: Probot) {
             issueOrPRNumber,
             context.payload.label?.name
           );
+          lock.release();
```

Comment:
Might be nicer to wrap the main logic into a helper, the acquire and release the lock around calling the helper. I could this being easy to add a logic branch and forget to release the lock.

Reply:
Good point. Will do.

Reply:
Done